Dockerfile-mongo-seed

```diff
@@ -0,0 +1,13 @@
+FROM  mongo:3.2
```

Comment:
The database is currently at 3.0.7, is there still a valid dockerfile for that one?